lib/services/local/dynamic_access.go

```diff
@@ -445,10 +445,10 @@ func itemFromAccessRequest(req types.AccessRequest) (backend.Item, error) {
 	if err != nil {
 		return backend.Item{}, trace.Wrap(err)
 	}
+	// Access requests are expired explicitly.
 	return backend.Item{
 		Key:      accessRequestKey(req.GetName()),
 		Value:    value,
-		Expires:  req.Expiry(),
```

Comment:
I can see that we're setting Expires field below in `itemFromAccessListPromotions` and `itemToAccessRequest` as well. What are those used for and how are they going to interact with changes in this PR? Do they need to be updated too?

Reply:
Have removed the setting of the expires field in itemToAccessRequest. For the itemFromAccessListPromotions the Expiry it calls will grab the explicit expiry field instead of the metadata one and will expire based on that. If we want the promotions to definitely expire at the same time as the relevant access request I can add that in but since we already just let them expire automatically I think it should be fine.